logstash-core/spec/logstash/pipeline_spec.rb

```diff
@@ -440,6 +447,10 @@ class TestPipeline < LogStash::Pipeline
     let(:settings) { LogStash::SETTINGS.clone }
     subject { LogStash::Pipeline.new(config, settings, metric) }
 
+    after :each do
```

Comment:
This can just be `after do ...` because `after :each` is default behavior of `after` without arguments.